docs/examples/entur/build-config.json

```diff
@@ -9,7 +9,6 @@
   "staticParkAndRide": true,
   "staticBikeParkAndRide": true,
   "maxDataImportIssuesPerFile": 1000,
-  "maxInterlineDistance": 200,
```

Comment:
I hope it's fine I removed this from the norway config examples as it's not used for netex as far as I can tell.